core/src/main/java/hudson/cli/Connection.java

```diff
@@ -192,6 +194,7 @@ public KeyAgreement diffieHellman(boolean side, int keySize) throws IOException,
      * @return
      *      A new {@link Connection} object that includes the transport encryption.
      */
+    @SuppressFBWarnings(value = "STATIC_IV", justification = "Not used. We should just remove it.")
```

Comment:
Perhaps start deprecating this if we should remove it

Reply:
The class has been deprecated for almost a year.